cmd/enterprise-portal/internal/samsm2m/samsm2m.go

```diff
@@ -0,0 +1,99 @@
+package samsm2m
+
+import (
+	"context"
+	"net/http"
+
+	"connectrpc.com/connect"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	otelcodes "go.opentelemetry.io/otel/codes"
+	"go.opentelemetry.io/otel/trace"
+
+	"github.com/sourcegraph/log"
+
+	sams "github.com/sourcegraph/sourcegraph-accounts-sdk-go"
```

Comment:
```suggestion
	"github.com/sourcegraph/log"
	sams "github.com/sourcegraph/sourcegraph-accounts-sdk-go"
```

I think we can group these together

Reply:
Auto-formatter won't allow it 😆 it keeps adding a space here